### ISSUE TITLE
Menu does not work correctly when using internet explorer 11 with touch

### ISSUE THREAD

**Issue by spy14** [SiCKRAGETV/sickrage-issues#1097]
BRANCH: (master)
COMMIT: (ff7aee90fd26db93cc571be1cb5c62a6fcf08e19) 
Your operating system: Windows 8.1, server on: Debian GNU/Linux 7
python version: Python 2.7.3
What you did: Clicked the menu
What happened: Nothing
What you expected: The menu to appear, see video

Video: I'm using a Dell Venue 8 pro tablet with windows 8.1. First chrome works as it should, after that you see the menu not working with internet explorer. 
https://www.youtube.com/watch?v=xu7bjtLmn-I

**Comment by fernandog** [comment]
@spy14 what if you click&hold?

**Comment by spy14** [comment]
click and hold opens the mouse right click menu

**Comment by fernandog** [comment]
@spy14 maybe some Javascript incompatibility

**Comment by abeloin** [comment]
@spy14 From [this document on Microsoft website](https://msdn.microsoft.com/en-us/library/ie/dn265029%28v=vs.85%29.aspx) you need to "Press and Hold" to simulate the hover effect then release on the menu item you want.

**Comment by spy14** [comment]
If I press and hold, the menu opens. If I release it on the top menu, the right click menu opens. If I try to slide it down and release on a menu item, the menu disappears or if I'm fast enough nothing happens.

**Comment by fernandog** [comment]
@spy14 still an issue with latest IE updates?

**Comment by spy14** [comment]
I updated my tablet to windows 10, so I can only test with Edge. 
I'm on BRANCH: (master) / COMMIT: (0308a49dcb9bf9bef03ec49a897a8373b399398c) 

The dropdown menus (like Manage, Logs & Errors) are still like in my old video. They stay open on Chrome but not on Edge.

**Comment by fernandog** [comment]
@MGaetan89  is there a way to make top menu to show menu items with hover AND click at the same time?  I think this will fix this issue

**Comment by MGaetan89** [comment]
I guess it's possible. But I think that setting two events on the same target is not right and may introduce an other issue on an other browser/version.
The best option to address both web and mobile would be to use only click to open the dropdown menu.

**Comment by fernandog** [comment]
I agree

@SiCKRAGETV/moderators agree?

**Comment by miigotu** [comment]
You could set one when only touch is available and the other when mouse is available ^

Browser should tell you these things.

**Comment by OmgImAlexis** [comment]
@MGaetan89 if you want to look into this then I'd suggest we use something like http://benhowdle.im/touche/ to make touch events pretend to be click events. That way there shouldn't be any reason to change anything.

**Comment by MGaetan89** [comment]
Why not simply switching to a traditional click event instead of hover?

**Comment by OmgImAlexis** [comment]
I thought the issue was with click and touch? It's 2 in the morning here. Maybe I need to reread this.

**Comment by MGaetan89** [comment]
The issue, from my understanding, is that the menu opens when you hover it with your mouth. But that event is not properly handled on touch screen. So, currently, clicking it does not work, and hovering it is tricky.

**Comment by fernandog** [comment]
Still have this issue?
I dont have it in my tablet with chrome

**Comment by OmgImAlexis** [comment]
@fernandog it's a IE issue.

**Comment by spy14** [comment]
Yes, still an issue with IE11 and Edge too.

**Comment by VinceVal** [comment]
Hi guys does SiCKRAGETV/SickRage#2713 help with this at all? With the split buttons, you should be able to tap/click the caret to toggle the dropdown.

**Comment by fernandog** [comment]
@spy14

**Comment by spy14** [comment]
The new dropdown is still not good with internet explorer and touch

**Comment by miigotu** [comment]
@spy14 you just tap the down arrow.

**Comment by spy14** [comment]
Yes, when I tap the arrow it opens for a half second and closes itself.

**Comment by OmgImAlexis** [comment]
@miigotu we need to either remove the hover to dropdown plugin or use js to determine if it's a touch device and disable it.

**Comment by spy14** [comment]
Sorry for shit quality.
https://www.youtube.com/watch?v=MPkWQxcCNw8&feature=youtu.be

**Comment by OmgImAlexis** [comment]
@spy14 in your SickRage install you'll find this file `gui/slick/js/_bower.min.js`, can you please replace it with the contents from [this](https://gist.githubusercontent.com/OmgImAlexis/d806a18282dd8930e951/raw/ad486d4f2b056a869b4f6634de074ed13dbba133/gistfile1.txt) page and let me know if it fixes anything. You may need to clear your cache to have the file update.

**Comment by spy14** [comment]
It is working now with touch, but there is no hover to dropdown on desktop.

**Comment by OmgImAlexis** [comment]
Yep, that plugin may need to be removed or setup to be disabled when touch is detected. That's what's causing the hover/touch issue.

**Comment by miigotu** [comment]
@OmgImAlexis cant it just be disabled on the right side of the split button?

**Comment by OmgImAlexis** [comment]
@miigotu the buttons we're using aren't meant to even really be used in the nav. Also in Bootstrap 4 it's all fixed so maybe we just disable this for now and in the Bootstrap 4 update we can add it back?

**Comment by miigotu** [comment]
yeah thats fine, I just liked auto-open :`(

**Comment by OmgImAlexis** [comment]
@miigotu we could probably add a css hover thing. It's just the js stuff breaks it really easily when you add touch into the mix.

**Comment by alisonatwork** [comment]
Would some guys like to test this PR to see if it behaves like you expect in Chrome (Android/Windows) and Safari (iOS)? Works nicely now in IE/Edge and Windows Phone.

**Comment by alisonatwork** [comment]
Okay i have been messing around a little more with this, and it still doesn't quite work. It does when the window is small enough that you switch to "mobile" mode (e.g. when you are in portrait mode or snapped across half the landscape screen), but the touch-tap still doesn't open the menu if you are in full-width landscape mode. I'll see if i can find a decent solution when i get some more time, perhaps we we can remove the data-hover attribute if it detects a touchscreen (though this will break 2-in-1s when you have the mouse attached). Argh.
